sources/Application/UI/Base/Component/DataTable/DataTableUIBlockFactory.php

```diff
@@ -180,6 +183,46 @@ protected static function RenderDataTable(DataTable $oDataTable, string $sStyle,
 		return $oContainer;
 	}
 
+	/**
+	 * Make a row actions toolbar template.
+	 *
+	 * @param iUIBlock $oTable datatable object that needs to use tTableRowActions trait
+	 *
+	 * @return \Combodo\iTop\Application\UI\Base\Component\Template\Template
+	 * @throws \Exception
+	 * @since 3.1.0
+	 */
+	public static function MakeActionRowToolbarTemplate(iUIBlock $oTable)
+	{
+		// test trait
+		$sTableClass = get_class($oTable);
+		if (!Utils::IsClassUsesDeepTrait($sTableClass, tTableRowActions::class)) {
```

Comment:
```suggestion
		if (!utils::IsClassUsesDeepTrait($sTableClass, tTableRowActions::class)) {
```

The `Utils` class with a capital "U" is not the class from iTop core, it's from another collector or extension. It might work but it can be misleading when doing "grep" on the codebase.